convert/src/vertex_to_bd/vertex_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,94 @@
+import tempfile
+from pathlib import Path
+from typing import Optional
+
+from .badgerdoc_format.annotation_converter import AnnotationConverter
+from .badgerdoc_format.badgerdoc_format import BadgerdocFormat
+from .badgerdoc_format.pdf_renderer import PDFRenderer
+from .badgerdoc_format.plain_text_converter import (
+    PlainTextToBadgerdocTokenConverter,
+)
+from .models import LabelStudioModel, S3Path
+
+
+class VertexToBDConvertUseCase:
+    def __init__(
+        self,
+        s3_client=None,
+    ) -> None:
+        page_border_offset = 15  # TODO: make as parameter
```

Comment:
Is TODO intended here?

Reply:
https://github.com/epam/badgerdoc/issues/171